lib/client-transport.js

```diff
@@ -39,7 +39,7 @@ function TAmqpTransport(client, _options,callback) {
     endpoint:  options.endpoint ||  'amqp://localhost',
     queueName: options.queueName || 'lycam-rpc',
     fromName: options.fromName || 'Unknown Client',
-
+    tempQueuePrefix: options.tempQueuePrefix || 'lycam-amq'
```

Comment:
Fun fact, you can't create a queue starting with amq as it is reserved (I tried), so defaulting to lycam-amq instead.